.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 .DS_Store
 /test/.mssql.json
 /test.js
-.vscode
\ No newline at end of file
+.vscode
+.idea
```

Comment:
FYI - this kind of thing shouldn't really be in a project's `.gitignore` - you should use a global ignore on your machine for your particular environment: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Dunno if it's a common practice. I've seen many open-source repos with at least .vscode ignores.

Reply:
I know, but they shouldn't ;)

it should be relevant to the project code not some person's choice of IDE.

Reply:
1) No harm done by having more ignores;
2) You can't expect everyone to have proper setup, so this increases chances of garbage being committed;

So while conceptually you are right, from pragmatic perspective I still think that these ignores make sense.

Reply:
I'm a maintainer of ~250 OSS repos, I'm happy to continue to educate people as part of my role of a maintainer.

Just to be clear (because this stuff is easily lost via text communication) I appreciate your opinion, your effort on this and your passion. So thanks for that, but if we let people just push arbitrary ignores under the guise that there's "no harm to more ignore rules" then I really don't think we're heading in a maintainable direction. The next step would just to use whitelisting in the `.gitignore` instead and that would be mad

Reply:
My intention wasn't to convince you that this change should be done in this particular case, I apologize if it sounded as I am trying to push my opinion through. I realize that every maintainer has his own style and opinion and I understand your line of reasoning.

Reply:
But yeah, I appreciate the discussion; while I not necessarily share your opinion, understanding it better was definitely useful :)

Reply:
No, not at all, I didn't feel like it came across like that at all - I just wanted to clarify that I wasn't trying to be unreasonable.

Thanks for the understanding :)